README.md

```diff
@@ -11,6 +11,8 @@ Environment variables that can be customized
 |--------------------|---------|----------------|----------------------|  
 | HAPI_JS_INFO_HOST  | string  | '0.0.0.0'      | HapiJS host binding  |  
 | HAPI_JS_INFO_PORT  | number  | 3000           | HapiJS port binding  |  
+| HAPI_JS_INFO_RECENT_PATH | string | '/recent' | Path to get recent requests |
+| HAPI_JS_INFO_NUM_RECENT_REQUESTS | number | 10 | Number of recent requests to cache |
```

Comment:
Very nitpicky... I know that markdown will convert this to a good looking table, but we should either justify the columns in the markdown source code for all of the rows or none at all.